src/main/java/io/openliberty/tools/common/plugins/util/AbstractContainerSupportUtil.java

```diff
@@ -51,6 +51,11 @@ public abstract class AbstractContainerSupportUtil {
      */
     public abstract void info(String msg);
 
+    public void setIsDocker(boolean isDocker) {
+        this.isDocker = isDocker;
+        this.checkedContainerType = true;
+    }
```

Comment:
Shouldn't the `checkPodmanVersion` method be called here if `isDocker` is false? Could call `checkDockerVersion` if `isDocker` parameter is true too.